client/src/i18n/index.js

```diff
@@ -16,6 +16,21 @@ const languagesMap = {
   nl
 }
 
+/**
+ * Return best match for lang and variant.
+ * @param {string} language string from navigator configuration or cookie.
+ * @returns the translation dict
+ */
 export function getTranslations(language) {
```

Comment:
Could be unnecessary, but just to protect against `language` being undefined and `split` throwing an error, I would suggest:

```suggestion
export function getTranslations(language = "") {
```